apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.jsx

```diff
@@ -559,27 +580,22 @@ const SwapForm = () => {
               countdown={!swapError && !idleState}
               showNoQuoteDexRate={showNoQuoteDexRate}
             />
-
-            {currentBanner === "LOGIN" ? (
```

Comment:
Just linting changes? @cjordan-ledger

Reply:
Just tested one of the banners and seems to work okay now that these are only rendered when `currentBanner` is set... rather than when `currentBanner` is set AND `showDetails && !showEmpty` is truthy.